docs/reference/errors-and-warnings/NU1302.md

```diff
@@ -0,0 +1,22 @@
+---
+title: NuGet Error NU1302
+description: NU2302 Error code
+author: Nigusu-Allehu
+ms.date: 07/25/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU1302"
+---
+
+# NuGet Warning NU1302
+
+<pre>You are running the 'restore' operation with an 'HTTP' source: myHttpSource. NuGet requires HTTPS sources. To use an HTTP source, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere for more information.

```

Comment:
The docs bot provides a preview link. Click it, and you'll see this is pretty much unreadable since the line is too long.

I know we use this pattern of code/pre blocks to show the example messages, but I think we should do better and make sure the docs are readable by default.